recipes/xkbcommon/all/conanfile.py

```diff
@@ -76,12 +76,12 @@ def validate(self):
             raise ConanInvalidConfiguration(f"{self.ref} is only compatible with Linux and FreeBSD")
 
     def build_requirements(self):
-        self.tool_requires("meson/1.1.0")
+        self.tool_requires("meson/1.2.2")
         self.tool_requires("bison/3.8.2")
         if not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
-            self.tool_requires("pkgconf/1.9.3")
+            self.tool_requires("pkgconf/2.0.3")
         if self._has_build_profile and self.options.get_safe("with_wayland"):
-            self.tool_requires("wayland/1.21.0")
+            self.tool_requires("wayland/<host_version>")
             self.tool_requires("wayland-protocols/1.31")
```

Comment:
cannot use `<host_version>` foy wayland-protocols, because it either either a requires or a tool_requires. Never both.